cmd/genji/shell/command.go

```diff
@@ -313,21 +313,21 @@ func runDumpCmd(db *genji.DB, tables []string, w io.Writer) error {
 
 	for i, table := range tables {
 		err = dumpTable(tx, table, w)
-		switch err {
-		case nil:
-			// Blank separation between tables.
-			if i > 0 {
-				if _, err := fmt.Fprintln(w, ""); err != nil {
-					return err
-				}
+		if err != nil {
+			// If table doesn’t exist we skip it.
+			if errors.Is(err, database.ErrTableNotFound) {
+				continue
 			}
-
-		case fmt.Errorf("%w: %q", database.ErrTableNotFound, table): // If table doesn't exist we skip it.
-			continue
-		default:
-			_, err = fmt.Fprintln(w, "COMMIT;")
+			_, err = fmt.Println(w, "COMMIT;")
```

Comment:
Oops, should have been `Fprintln` here since we are writing to an `io.Writer`.

```suggestion
			_, err = fmt.Fprintln(w, "COMMIT;")
```